ngmix/gmix.py

```diff
@@ -684,15 +684,31 @@ def make_galsim_object(self, Tmin=0.001):
             Minimum size for gaussians.  Galsim doesn't allow objects
             with less than zero size because when convolving it renders
             the object
+        gsparams: GSParams or dict, optional
+            A GSParams object (or dict convertable to one) that sets
+            certain useful parameters for GalSim renderings (e.g.
+            a larger maximum FFT size)
         """
+
         import galsim
 
+        if gsparams and (type(gsparams) is not galsim._galsim.GSParams):
+            if isinstance(gsparams, dict):
+                # Convert to actual gsparams object
+                gsparams = galsim.GSParams(**gsparams)
+            else:
+                raise TypeError('Only `dict` and `galsim.GSParam` types allowed'
+                                ' for gsparams; input has type of {}.'
+                                .format(type(gsparams)))
+
```

Comment:
where is gsparams being used?

Reply:
Ah shoot, left out the most important line! It is passed as an optional parameter to `galsim.Gaussian()`. Will add that.